src/models/index.ts

```diff
@@ -22,6 +23,7 @@ export interface PersistentState {
   bookmark: Bookmark;
   config: VoyagerConfig;
   log: Log;
+  relatedViewToggler: RelatedViewToggler;
```

Comment:
Perhaps just `relatedViews` -- as we may have other related views feature in the future that are not boolean / togglers.